### DISCUSSION TITLE
*ngFor does not work on carousel

### DISCUSSION THREAD

**Discussion by AlKYala** [ivylaboratory/angular-carousel#59] [Q&A] [answered]
I tried to iterate over an array of objects to include images in the carousel and it does not work for me.

```
   <carousel>
      <ng-container *ngFor="let marke of this.marken">
        <div class="carousel-cell">
          <img [src]="marke.markenPicUrl">
        </div>
      </ng-container>
    </carousel>
```

And 

```
    <carousel>
      <ng-container *ngFor="let marke of this.marken">
        <div class="carousel-cell">
          <img src="{{marke.markenPicUrl}}">
        </div>
      </ng-container>
    </carousel>
```

Dont work.

**Comment by drozhzhin-n-e** [comment, marked as answer]
What error are you getting?

**Reply by AlKYala** [reply]
No error on the console but if I loop over the elements the cards without carousel the elements show.

**Reply by drozhzhin-n-e** [reply]
try it

```
<carousel>
	<div class="carousel-cell" *ngFor="let marke of this.marken">
		<img [src]="marke.markenPicUrl">
	</div>
</carousel>
```

**Reply by AlKYala** [reply]
Thank you, this works. And I've found another bug - the elements in the carousel are only shown if I resize the window (chrome). Is this a known bug?

**Reply by siddharth318** [reply]
Hi AlKYala,
I am facing the same resize bug, did u find a fix?

**Comment by abu18saud** [comment]
Any update about this question?
